base_layer/core/src/proof_of_work/monero_rx/helpers.rs

```diff
@@ -51,6 +51,7 @@ pub fn monero_difficulty(header: &BlockHeader, randomx_factory: &RandomXFactory)
 
 fn get_random_x_difficulty(input: &[u8], vm: &RandomXVMInstance) -> Result<(Difficulty, Vec<u8>), MergeMineError> {
     let hash = vm.calculate_hash(&input)?;
+    debug!(target: LOG_TARGET, "RandomX Hash: {:?}", hash);
```

Comment:
```suggestion
    debug!(target: LOG_TARGET, "RandomX Hash: {:?}, seed:{}", hash. vm.seed);
```

Reply:
That is contained in the log line above this already